### ISSUE TITLE
Diablo fight PM3-5 COP missions

### ISSUE THREAD

**Issue by Vorpall** [HomepointXI/Issues#514]
**Date & Time**: 

9:02 pm

**Client Version (use `/ver` in game):**

30191204_1

**Character Name:**

Vorpal

**Discord Name (if different from in game):**

Vorpal

**Nation:**

San d'Oria

**Job(level)/Sub Job(level):**

Blu/rdm

**NPC or Monster or item Name:**

Diablo 

**Zone name:** 

battlefield 

**Coordinates (use `!where`):**

n/a

**Item tool link (for items issues only):**

n/a

**Multi-boxing? (multiple clients on same connection):**

No

**Steps to reproduce:**
when i fall down  diablo disappear and i fight only 6 Diremite

**Any additional information:**

**Comment by EpicTaru** [comment]
I watched you do this fight. When you drop down onto the tiles to face Diabolos, you just fall right thru the tiles a second after you land on them, Diabolos is still present above the tiles.  You see the diremites because those are in the pit below the tiles.  I also noticed that if you aggro Diabolos from the ledge and get pulled down to him, you don't fall thru the tiles immediately.

The tiles are basically "doors" that "open" at set intervals, causing anyone located on an "open" tile to fall thru them and into the pit.  By default, the tiles should all be closed but that doesn't appear to be the case right now.

Until this issue is resolved, that is probably the workaround to use....just aggro Diabolos and let him pull you down to him.
